mplang/v2/libs/device/api.py

```diff
@@ -462,10 +473,18 @@ def _ensure_tee_session(
     v_sk, v_pk = simp.pcall_static((frm_rank,), crypto.kem_keygen, _TEE_KEM_SUITE)
     v_pk_at_tee = simp.shuffle_static(v_pk, {tee_rank: frm_rank})
```

Comment:
Can we ensure that shuffle_static uses a certified p2p channel? (e.g. https with certificate) Otherwise, TEE may not be able to verified the sender's pk is legit (man-in-the-middle attack)